routers/users.py

```
@@ -0,0 +1,257 @@
+            settings.SECRET_KEY,
+            algorithms=[settings.ALGORITHM]
+        )
+        return payload if payload.get("sub") else None
+    except JWTError:
+        return None
+
+DB_FILE = "./users.json"
+def get_db_users():
+    if not os.path.exists(DB_FILE):
+        return []
+    with open(DB_FILE, "r", encoding="utf-8") as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)